package/yast2-security.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Jun 12 16:27:53 UTC 2015 - ancor@suse.com
+
+- When checking services, systemd aliases are now taken into
+  account (so, for example, rsyslog is accounted and syslog).
```

Comment:
No fate number yet because there is no version bump yet :-) I plan to add the fate when the feature is finished and the version number is increased.